templates/docs/examples/patterns/grid/grid-breakout-right-aside.html

```diff
@@ -0,0 +1,27 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Grid / Wide - aside on the right{% endblock %}
```

Comment:
Please rename it, as it's not Grid example anymore ("Fluid breakout / Wide - aside on the right"?).

Also, because it's not a pattern move the file from `/templates/docs/examples/patterns/grid` to `/templates/docs/examples/layouts/...`.

Reply:
done